go/src/koding/kites/kloud/provider/marathon/testdata/multi-container.json.golden

```diff
@@ -98,7 +111,7 @@
           "health_check": [
             {
               "command": {
-                "value": "curl -f -X GET http://$$HOST:$${PORT_56789}/kite"
+                "value": "true"
```

Comment:
https://github.com/koding/koding/pull/9547/files#diff-12a6b9ca19c0ec2dc9242fd05655ce9eR26

The healtchecking is not needed to have working marathon provider - I wanted to be too correct to provide a healthcheck to ensure klient works. I need to both extend klient to allow for healthchecking (`curl .../kite` is not enough as it does not fail when klient has e.g. broken tunnel connection) and extend kloud provider for marathon with http healthchecks, but I'd need to map the ports for that.